StripeFinancialConnections/StripeFinancialConnections/Source/Native/Success/SuccessFooterView.swift

```diff
@@ -51,20 +48,6 @@ final class SuccessFooterView: UIView {
         let footerButtonStackView = UIStackView()
         footerButtonStackView.axis = .vertical
         footerButtonStackView.spacing = 12
-        if didSelectLinkAnotherAccount != nil {
```

Comment:
should be easy to review just by reviewing code as we had this toggled off for some cases already